common/goal_distance_calculator/launch/goal_distance_calculator.launch.xml

```diff
@@ -3,6 +3,5 @@
   <arg name="config_file" default="$(find-pkg-share goal_distance_calculator)/config/goal_distance_calculator.param.yaml"/>
   <node pkg="goal_distance_calculator" exec="goal_distance_calculator_node" name="goal_distance_calculator" output="screen">
     <param from="$(var config_file)"/>
-    <param name="oneshot" value="$(var oneshot)"/>
```

Comment:
Should `oneshot` at line 2 above also be removed?

Reply:
Done.